kubernetes/overlays/prod/prompt-proto-service/prompt-proto-service.yaml

```diff
@@ -25,7 +25,14 @@ spec:
         - name: RUBIN_INSTRUMENT
           value: LATISS
         - name: PIPELINES_CONFIG
-          value: (survey="AUXTEL_PHOTO_IMAGING")=${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/ApPipe.yaml
+          value: >
+            (survey="AUXTEL_PHOTO_IMAGING")=${PROMPT_PROTOTYPE_DIR}/pipelines/${RUBIN_INSTRUMENT}/ApPipe.yaml
```

Comment:
Space delimited will work fine and I tested it locally by validating pod env value and helm passes it through.

Want to ask though does the pipelines config need to be space delimited or would a config file work (properties, yaml, etc..)?  The normal convention is to do a config map for file type values.

Reply:
The current code assumes it's an environment variable because I was not aware of config maps at the time. So there would need to be substantial changes made on that end.

My main requirement is that the configuration (which may change nightly or faster) should not put any additional burden on deployment; I think in the case of config maps that means the map update would need to be added to the existing `make apply` command? (There's also the question of how to handle the implicit instrument-dependence; a shared multidimensional mapping is probably better in the long run, but a per-instrument map would be easier to set up before the next run.)

Note that the final configuration may be quite complicated (e.g., `survey="AUXTEL_PHOTO_IMAGING"` and `gal_lat < 10` and `nimages <= 2`), so we can't assume a simple key-value mapping.

Reply:
Makes sense and agree with keeping approach.  Thanks.

Reply:
Would it be useful to have a transitional stage where the config map is just a single (multiline) string value, and that gets fed into `PIPELINES_CONFIG`? Would that provide any benefit for an eventual migration to a proper config file?